TCP_Server.py

```diff
@@ -5,6 +5,34 @@
 logging.basicConfig(level=logging.INFO, format='[%(asctime)s] %(message)s')
 
 
+class Connection:
+    def __init__(self, conn):
+        self.EOT = '\n\x04\n'
+        self.conn = conn
```

Comment:
I know it is late and I cling, but shouldn't these fields be private?

Reply:
more sources/code needs refactor - so.... i suggest another PR for private fields and unification of source notation.

Reply:
Ok, I trust you :)